### ISSUE TITLE
Please remove 2

### ISSUE THREAD

**Issue by zoujiaqing** [ldc-developers/ldc2.snap#9]
ldc.snap is good.

**Comment by WebDrake** [comment]
@zoujiaqing, thanks for your interest in the snap package.

The `2` is there for a reason: snap packages will 'namespace' any apps they expose that don't match the package name.  This is why in the current snap package, the `ldmd2`, `ldc-profdata` and `ldc-prune-cache` commands are all exposed as, respectively, `ldc2.ldmd2`, `ldc2.ldc-profdata` and `ldc2.ldc-prune-cache`.

So, if the package were named just `ldc`, the `ldc2` command would instead be `ldc.ldc2`.  I decided that was not a very nice situation to have to deal with (especially since many programs check if `ldc2` is the compiler by examining its name), so I named the package `ldc2`.

Can I ask you the favour of confirming whether the snap package works for you?  Are you able to successfully install it and compile programs with it?
